src/api/client.ts

```diff
@@ -81,6 +82,28 @@ export function getNostoClient(): PromiseLike<NostoClient> {
         }
     })
 }
+/**
+ * @group Nosto Client
+ * @category Core
+ */
+type LogLevel = 'error' | 'warn' | 'info' | 'debug'
+export function log(msg: string, error: unknown, level?: LogLevel): void
+export function log(msgOrError: string | unknown, level?: LogLevel): void
+/**
+ * Logs an error to Nosto and the console
+ * @param msgOrError string message or error instance
+ * @param errorOrLevel error instance or log level string
+ * @param optLevel optional log level string
+ */
+export function log(msgOrError: string | unknown, errorOrLevel: LogLevel | unknown, optLevel?: LogLevel) {
+   const msg = typeof msgOrError === "string" ? msgOrError : undefined
+   const error = optLevel ? errorOrLevel : (!msg ? msgOrError : undefined)
+   const level = (optLevel || (typeof errorOrLevel === "string" ? errorOrLevel : undefined)) as LogLevel
+   getNostoClient().then(api => {
+        api.captureError(error ?? new Error(msg), 'nostoAutocomplete', level)
```

Comment:
I'm not sure about this if we need to log msg if error is undefined, but I suppose either one of them will be defined